src/components/App/App.js

```diff
@@ -0,0 +1,87 @@
+import { useState, useEffect } from 'react';
+import { Switch, Route } from 'react-router-dom';
+import Signup from '../Signup/Signup';
+import Signin from '../Signin/Signin';
+
+import {
+  MINLENGTH,
+  validEmailRegexp,
+  validPasswordRegexp,
+} from '../../utils/constants';
+
+function App() {
+  const [isBtnDisabled, setIsBtnDisabled] = useState(true);
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
```

Comment:
One little suggestion here, 
- name
- email
- password

Could be saved in one useState just as errors, something like: 
`const [userData, setUserData] = useState({name: '', email: '', password: ''})`